apps/mosaico/src/Mosaico/Eval.purs

```diff
@@ -0,0 +1,15 @@
+module Mosaico.Eval where
+
+import Prelude
+
+import Effect (Effect)
+import Effect.Uncurried (EffectFn1, runEffectFn1)
+
+foreign import evalExternalScriptsImpl :: EffectFn1 (Array String) Unit
+
+evalExternalScripts :: Array ScriptTag -> Effect Unit
+evalExternalScripts scriptTags = runEffectFn1 evalExternalScriptsImpl $ map (\(ScriptTag scriptTag) -> scriptTag) scriptTags
```

Comment:
Like I said for `Mosaico.purs`, I'd prefer adding `<script></script>` here.

Also: I looked at how the javascript function is implemented and it's bonkers. It creates an element, inserts this `<script>`-wrapped thing into it, then grabs its `innerHTML` and then calls `eval` on that. Why isn't it just calling `eval` on the string in the first place without creating some "dummy" element for it? But let's leave it like that for now, I guess this has been waiting for long enough as is.

Reply:
> Why isn't it just calling eval on the string in the first place

If we only had the script as a string, this would work. However we are getting strings of script tags, such as `<script>some js</script>`, so we need a way to export that javascript and evaluate it. AFAIK it needs to be done in a silly fashion like inserting it as inner html somewhere first. Also, we might get script tags without inner html, but with a `src` attribute `<script src="/script.js"></script>` where we again need to make it into an element first, fetch the script and evaluate it.

This is designed to work with whatever Lettera gives us as `externalScripts` (full script elements), but of course here we only have the inner html, so we could have another function that `evals` the strings it gets directly.

Reply:
Or we could use the `HtmlRenderer`I'm developing

Reply:
Would it be possible to keep the code like this for now and maybe later do these changes?